charts/frontend/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v2
 name: frontend
 description: A Helm chart for GOV.UK Frontend
 type: application
-version: 0.6.2
+version: 0.7.0
```

Comment:
Normally this would be a major version bump because I'm removing stuff from the public interface, but I've made it a minor because going to 1.0 for this would be even more misleading IMO.